doc/architecture/architecture_overview.adoc

```diff
@@ -48,4 +52,4 @@ However, this does not mean that it is optional to fill the field.
 For the purpose of providing a complete interface, all existing fields should be set, unless not setting a field carries a specific meaning, as indicated in the accompanying comment.
 
 NOTE: All field numbers equal to or greater than 10000 are available for user-specific extensions via custom fields.
-No future evolution of OSI will therefore use field numbers equal to or greater than 10000.
\ No newline at end of file
+No future evolution of OSI will therefore use field numbers equal to or greater than 10000.
```

Comment:
What has changed here? Or is this highlighting some bug in the diff-function from GitHub?

Reply:
Probably some CRLF/LF stuff? I'm working on some documentation additions on the StreamingUpdate message right now anyway. I'll try deleting and adding the last line. Maybe that does the trick.